### PR TITLE
pull request for TSE review

### PR DESCRIPTION
**Pull request by muhenge**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1704 @@
...
+                    <a href="#"><img class="h-75 w-100" src="images/Newsweek_Logo.svg.png" alt="logo"></a>
+                </div>
+                <div
+                    class="col-8 col-sm-8 col-md-5 col-lg-4 d-flex d-md-flex d-sm-flex flex-row-reverse align-items-center align-items-sm-center vertical-align-middle">
+                    <button class="navbar-toggler text-white d-lg-none  " type="button" data-toggle="collapse"
+                        data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent"
+                        aria-expanded="false" aria-label="Toggle navigation">
+                        <span class="navbar-toggler-icon text-white">...</span>
+
+                    </button>
```
Comment (suretrust) [inline index.html]: ##### Navigation bar
* [x] **It changes with RWD breakpoints** ([link to page](https://www.newsweek.com/)). Your navbar is not properly sized and contains sub navbar on the medium screen.

**Yours**
Screenshot 2020-01-03 at 17 12 15


**Original**
Screenshot 2020-01-03 at 17 11 27

File: index.html [hunk 2]
```diff
@@ -0,0 +1,1704 @@
...
+                    <div class="col-12 col-md-9">
+                        <div class="divider w-100 d-flex justify-content-center">
+                            <div class="newsweek-logo-s"></div>
+                            <span>IN THE MAGAZINE</span>
+                        </div>
+                        <div class="row">
+                            <div class="col-12 col-md-4">
+                                <article class="magazine-stories">
+                                    <a href="#"><img class="img-magazine"
+                                            src="https://d.newsweek.com/en/full/1548831/fe-csr-01-1041330598-banner.webp?w=466&h=311&l=52&t=49&q=90&f=e6d8a68693cdcf8906bbbe92a804bc55"
```
Comment (suretrust) [inline index.html:1155]: The "IN THE MAGAZINE" section is messy on a medium screen. [Link to page](https://www.newsweek.com/).

**Yours**
Screenshot 2020-01-03 at 17 15 48

**Original**
Screenshot 2020-01-03 at 17 16 06

File: index.html [hunk 3]
```diff
@@ -0,0 +1,1704 @@
...
+                                                src="https://d.newsweek.com/en/full/1551658/sneeze-cold-illness-flu-stock-getty.webp?w=397&h=265&f=e539b9b52e5961febb6e31862bdd673b"
+                                                alt="" /></a>
+                                        <span class="badge badge-light">WORLD</span>
+                                    </div>
+                                    <div class="card-body">
+                                        <h2 class="card-title">
+                                            <a class="links font-weight-bold" href="#">
+                                                Lorem ipsum dolor sit</a>
+                                        </h2>
+
```
Comment (suretrust) [inline index.html]: ##### Footer

* [ ] It contains social media icons ([link to page](https://www.newsweek.com/))
* [ ] **It changes with RWD breakpoints** ([link to page](https://www.newsweek.com/))

**Review by suretrust — CHANGES_REQUESTED** [review]
Hi @muhenge,

### Great job thus far. 🙌 Please see my comments on your code.

**Kindly submit for another code review once these issues have been resolved.** _Happy coding!_ 🎉

**Comment by izaiasneto4** [comment]
#### Hi @muhenge. Great job implement all of the required changes. Status: Approved :+1: 

TSE: Izaias Neto

**Comment by izaiasneto4** [comment]
### Hi @muhenge. Nice job implementing the requested changes, there's just one thing left to fix before we can get your project approved.  :arrow_down: 

- [ ] Let's fix all of the errors given by the [validator](https://validator.w3.org/nu/#file)

Please, when finished submit the project for a new request. :pray:

**Review by mosesogwo — APPROVED** [review]
### :trophy: Congratulations! You have done well.

Please consider aligning the three columns at the top properly before merging to master. Since this wasn't pointed out before now, I'll give my approval.

image


Cheers.

### Status: Approved

**Pull request merged**